client/src/components/add-employer-component/job-lead-component/index.jsx

```diff
@@ -0,0 +1,278 @@
+  };
+  const [open, setOpen] = useState(false);
+
+  const handlePageChange = (event, value) => {
+    onPageChange(value);
+  };
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleBackButtonClick = () => {
+    onPageChange(2);
+  };
+
+  const handleAddJobLead = () => {
+    const newId = employerData.length;
+    setEmployerData([
+      ...employerData,
+      {
+        id: newId,
+        jobTitle: "",
+        compensation: "",
+        hoursPerWeek: "",
+        description: "",
+        creationDate: null,
+        expirationDate: null,
+        employmentType: "",
+      },
+    ]);
+  };
+
+  const handleInputChange = (input, id, field) => {
+    const updatedJobLeads = employerData.map((lead) =>
+      lead.id === id ? { ...lead, [field]: input } : lead,
+    );
+    setEmployerData(updatedJobLeads);
+  };
+
+  const handleResetInputs = () => {
+    resetInitialState();
+  };
+
+  return (
+    <Container>
+      <H1>Adding a new Employer</H1>
+      <Body>
+        Input information about any job leads associated with the employer.
+      </Body>
+      {employerData.map((lead) => (
+        <JobLeadContainer key={lead.id}>
+          <H3>Job Lead</H3>
+          <TextField
+            fullWidth
+            sx={{ m: 1, width: "96%" }}
+            id={`jobTitle${lead.id}`}
+            value={lead.jobTitle}
+            onChange={(input) =>
+              handleInputChange(input.target.value, lead.id, "jobTitle")
+            }
+            label="Job Title"
+          />
+          <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+            <InputLabel id="compendationLabel">Compensation</InputLabel>
+            <OutlinedInput
+              id={`compensation${lead.id}`}
+              startAdornment={
+                <InputAdornment position="start">$</InputAdornment>
+              }
+              endAdornment={
+                <InputAdornment position="end">/hour</InputAdornment>
+              }
+              label="Compensation"
+              value={lead.compensation}
+              onChange={(input) =>
+                handleInputChange(input.target.value, lead.id, "compensation")
+              }
+            />
+          </FormControl>
+          <TextField
+            fullWidth
+            sx={{ m: 1, width: "47%" }}
+            id={`hoursPerWeek${lead.id}`}
+            label="Hours per week"
+            variant="outlined"
+            value={lead.hoursPerWeek}
+            onChange={(input) =>
+              handleInputChange(input.target.value, lead.id, "hoursPerWeek")
+            }
+          />
+          <TextField
+            fullWidth
+            sx={{ m: 1, width: "96%" }}
+            id={`description${lead.id}`}
+            label="Job Description"
+            multiline
+            rows={4}
+            value={lead.description}
+            helperText="*Required"
+            onChange={(input) =>
```

Comment:
rename input in each of the methods triggered by an onchange event to e or event